commands/create_command.py

```diff
@@ -0,0 +1,30 @@
+from Icommand import ICommand
+import os, json, sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from SchemaKeys import SchemaKeys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self.__validate_schema(schema)
+        self.data = json.load(open(schema, "r"))
+
+    def __validate_schema(self, schema):
+        if (schema == None or not os.path.exists(os.path.join(parent_dir, schema))):
+            raise Exception("FileNotFound")
+        data = json.load(open(schema, "r"))
```

Comment:
You are loading schema two times, you can rename method by `__validate_and_load_Schema (schema_path)`